engine/vcs/github/client_pull_request.go

```diff
@@ -140,3 +94,80 @@ func (g *githubClient) PullRequestComment(ctx context.Context, repo string, id i
 
 	return nil
 }
+
+func (g *githubClient) PullRequestCreate(ctx context.Context, repo string, pr sdk.VCSPullRequest) (sdk.VCSPullRequest, error) {
+	path := fmt.Sprintf("/repos/%s/pulls", repo)
+	payload := map[string]string{
+		"title": pr.Title,
+		"head":  pr.Head.Branch.DisplayID,
+		"base":  pr.Base.Branch.DisplayID,
+	}
+	values, _ := json.Marshal(payload)
+	res, err := g.post(path, "application/json", bytes.NewReader(values), &postOptions{skipDefaultBaseURL: false, asUser: true})
+	if err != nil {
+		return sdk.VCSPullRequest{}, sdk.WrapError(err, "Unable to post status")
+	}
+	defer res.Body.Close()
+
+	body, err := ioutil.ReadAll(res.Body)
+	if err != nil {
+		return sdk.VCSPullRequest{}, sdk.WrapError(err, "Unable to read body")
+	}
+
+	var prResponse PullRequest
+	if err := json.Unmarshal(body, &prResponse); err != nil {
+		return sdk.VCSPullRequest{}, sdk.WrapError(err, "Unable to unmarshal pullrequest %s", string(body))
+	}
+
+	return fromPullRequestToVCSPuyllRequest(prResponse), nil
+}
+
+func fromPullRequestToVCSPuyllRequest(pullr PullRequest) sdk.VCSPullRequest {
```

Comment:
could be more sexy if it's a method on `PullRequest` to call like this `prResponse.ToVCSPullRequest()` and by the way there is a mistake in the name of your function